### PR TITLE
Super call added to List field serialize method

### PR DESCRIPTION
**Pull request by rajeshkaushik**
This help to support "required" enabled on List field.

### CODE REVIEW

File: sanic_openapi/doc.py [hunk 1]
```diff
@@ -120,7 +120,8 @@ def serialize(self):
             items = serialize_schema(self.items[0])
         return {
             "type": "array",
-            "items": items
+            "items": items,
+            **super().serialize()
```
Comment (ahopkins) [inline sanic_openapi/doc.py]: Wouldn't you want to call `super().serialize` first, and then `update()` it with `type` and `items`? That way those keys do not accidentally get overwritten?

Comment (rajeshkaushik) [reply sanic_openapi/doc.py]: @ahopkins Makes sense. I just aligned with all other super calls for serialize method in different fields and kept is same. Let me know if you still want the super call as first statement or we can leave it for future when changes would be done for all fields?

**Comment by ahopkins** [comment]
I'll let @huge-success/sanic-open-api make that call. Was just an observation. I'll be honest I do not look too closely at the code so was unfamiliar that this is a normal pattern in the code base.

**Comment by chenjr0719** [comment]
Agree that call `super().serialize()` first would be more save. But, on the other hand, make the code base with the same style is also important.

I think we can make this PR go first and open another PR to handle the `super()` call in all fields.

**Pull request merged**